test/test-spawn.c

```diff
@@ -654,6 +654,7 @@ TEST_IMPL(spawn_tcp_server) {
   struct sockaddr_in addr;
   int fd;
   int r;
+  uv_os_fd_t handle;
```

Comment:
Sorry, that was me. Only tested that there was no warnings on Linux.
`handle`should only be defined on `Windows`